go.sum

```diff
@@ -170,6 +170,8 @@ github.com/hashicorp/go-slug v0.3.0/go.mod h1:I5tq5Lv0E2xcNXNkmx7BSfzi1PsJ2cNjs3
 github.com/hashicorp/go-sockaddr v0.0.0-20180320115054-6d291a969b86/go.mod h1:7Xibr9yA9JjQq1JpNB2Vw7kxv8xerXegt+ozgdvDeDU=
 github.com/hashicorp/go-tfe v0.3.16 h1:GS2yv580p0co4j3FBVaC6Zahd9mxdCGehhJ0qqzFMH0=
 github.com/hashicorp/go-tfe v0.3.16/go.mod h1:SuPHR+OcxvzBZNye7nGPfwZTEyd3rWPfLVbCgyZPezM=
+github.com/hashicorp/go-tfe v0.3.17 h1:4n4Z1Kr5hXKI2ntJLyso9yYFHjSBVYRGgK8BG48H6OQ=
+github.com/hashicorp/go-tfe v0.3.17/go.mod h1:SuPHR+OcxvzBZNye7nGPfwZTEyd3rWPfLVbCgyZPezM=
```

Comment:
It looks like you didn't do a `go mod tidy`

Reply:
Running `go mod tidy` has no effect on this file.

Reply:
I don't know what to say here except that I can't figure out how to fix this without manually editing the file.

Reply:
I found out why this is... It's because Terraform depends on `go-tfe` v0.3.16. So this is expected and correct 👍